utils/mil_msgs/msg/LabeledObject.msg

```diff
@@ -0,0 +1,11 @@
+# Represents a single object labeled from LabelMe
+
+string name                   # Name/Type of object entered into labeling program
+
+string attributes             # text entered into attribute box of labeling program
+
+geometry_msgs/Point[] polygon # vertices of polygon drawn in labeling program
+                              # X,Y represent pixel cordinates in labeled image
+                              # Z is ignored, should be 0
```

Comment:
I'm more interested in just modeling the raw data from LabelMe in ros with this. It isn't clear that all or even most of the tools using this will make use of a bounding rect.